### ISSUE TITLE
RichText placeholder text must have a sufficient color contrast ratio

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#9532]
See related #9502 

The `RichText` component, when it's empty, displays a placeholder text. This is not a real HTML placeholder attribute, it's just an element that mimics a placeholder. Regardless, the text color doesn't have a sufficient color contrast ratio to meet the WCAG level AA requirement.

Turns out the actual color used is also different across the blocks. For some blocks it's `#32373c` with an `opacity: 0.5` set on its wrapper. The resulting contrast ratio is a [too low 2.79:1](https://jdlsn.com/color/?type=hex&color=32373c&color2=ffffff&alpha=0.5).

For other blocks the color is slightly different, for example:

preformatted text 
`color: #23282d;`

verse 
`color: #191e23;`

Haven't checked all the blocks but it's likely there are other blocks where the color is slightly different. This depends on the actual markup used to render the fake placeholder, which varies depending on the block and thus inherits different CSS rules.

Regardless, the WCAG rule is very simple: for normal text, the color contrast ratio must be at least 4.5:1.

<img width="685" alt="screen shot 2018-09-01 at 13 37 43" src="https://user-images.githubusercontent.com/1682452/44945635-e673c000-aded-11e8-95f2-b669b74fbd2a.png">
